### ISSUE TITLE
KOTOR hangs just after character creation

### ISSUE THREAD

**Issue by RealDoigt** [xemu-project/xemu#620]
### Title

https://xemu.app/titles/4c410003/#Star-Wars-Knights-of-the-Old-Republic
USA/NTSC 1.1

### Bug Description

The game hangs on the load screen right after character creation.

I suspect any character choice will reproduce the result I got, but here's what I did in detail:
1. Create a new game
2. Choose soldier class
3. Choose a template character
4. Choose the mustachioed asiatic model
5. Name him 
6. Game is done and hung

### Expected Behavior

I was expecting the loading screen to be a temporary screen and get on the action eventually.

### xemu Version

Version: 0.6.2-22-gedbe932d8c
Branch:  master
Commit:  edbe932d8c4780f92665372e15f672874be5e93b
Date:    Thu Jan  6 09:11:16 UTC 2022

### System Information

Operating System: Kubuntu 21.04
Kernel Version: 5.11.0-44-generic
OS Type: 64-bit
Graphics Platform: X11
Processors: 4 × Intel® Core™ i5-4460 CPU @ 3.20GHz
Memory: 15.6 Gio of RAM
Graphics Processor: AMD OLAND

**Comment by Shoegzer** [comment]
Duplicate of #393.  Please close this one and report feedback over there, thanks.

**Comment by RealDoigt** [comment]
Sorry just got the email, I had a bad spam situation. My error is definitely different. I can start a game if I don't use a custom character, but not if I create a custom one. Totally different, you should read what I wrote.

**Comment by RealDoigt** [comment]
can you please reopen this issue @Shoegzer

**Comment by Shoegzer** [comment]
I did read what you wrote.  You said above "Choose a template character" which is not a custom character, and was in fact the base test case for my own issue report.  Just to be sure though, I just tested with a template AND custom character, and I go ingame on both, so the issue is on your end.  Perhaps try with a new hard drive image or redump your game.

Also, I cannot reopen this issue because you're the one who created it.  Although I strongly recommend not reopening it, I have no authority over issues you create.
